src/api/rest/providers/github/endpoints/search.js

```diff
@@ -0,0 +1,23 @@
+import { CALL_API } from 'api/rest/middleware';
+import { REPOS_BY_SEARCH } from 'api/rest/actions/search';
+import { handlePaginatedApi } from 'api/rest/providers/github';
+
+import Schemas from '../schemas';
+
+const _searchRepos = (query, nextPageUrl) => ({
+  id: query,
+  [CALL_API]: {
+    types: REPOS_BY_SEARCH,
+    endpoint: nextPageUrl,
+    schema: Schemas.REPO_ARRAY,
+    normalizrKey: 'items',
+  },
```

Comment:
Added `normalizrKey` here because the repositories array is in the `item` keys of the JSON response.
We may need to come up with something cleaner if there are more than a level to cross